client/app/visualizations/map/initMap.js

```diff
@@ -0,0 +1,188 @@
+import { isFunction, each, map } from 'lodash';
+import L from 'leaflet';
+import 'leaflet.markercluster';
+import 'leaflet/dist/leaflet.css';
+import 'leaflet.markercluster/dist/MarkerCluster.css';
+import 'leaflet.markercluster/dist/MarkerCluster.Default.css';
+import 'beautifymarker';
+import 'beautifymarker/leaflet-beautify-marker-icon.css';
+import markerIcon from 'leaflet/dist/images/marker-icon.png';
+import markerIconRetina from 'leaflet/dist/images/marker-icon-2x.png';
+import markerShadow from 'leaflet/dist/images/marker-shadow.png';
+import 'leaflet-fullscreen';
+import 'leaflet-fullscreen/dist/leaflet.fullscreen.css';
+import resizeObserver from '@/services/resizeObserver';
+
+// This is a workaround for an issue with giving Leaflet load the icon on its own.
+L.Icon.Default.mergeOptions({
+  iconUrl: markerIcon,
+  iconRetinaUrl: markerIconRetina,
+  shadowUrl: markerShadow,
+});
+
+delete L.Icon.Default.prototype._getIconUrl;
+
+const iconAnchors = {
+  marker: [14, 32],
+  circle: [10, 10],
+  rectangle: [11, 11],
+  'circle-dot': [1, 2],
+  'rectangle-dot': [1, 2],
+  doughnut: [8, 8],
+};
+
+const popupAnchors = {
+  rectangle: [0, -3],
+  circle: [1, -3],
+};
+
+const createHeatpointMarker = (lat, lon, color) => L.circleMarker(
+  [lat, lon],
+  { fillColor: color, fillOpacity: 0.9, stroke: false },
+);
+
+const createDefaultMarker = (lat, lon) => L.marker([lat, lon]);
+
+function createIconMarker(lat, lon, { iconShape, iconFont, foregroundColor, backgroundColor, borderColor }) {
+  const icon = L.BeautifyIcon.icon({
+    iconShape,
+    icon: iconFont,
+    iconSize: iconShape === 'rectangle' ? [22, 22] : false,
+    iconAnchor: iconAnchors[iconShape],
+    popupAnchor: popupAnchors[iconShape],
+    prefix: 'fa',
+    textColor: foregroundColor,
+    backgroundColor,
+    borderColor,
+  });
+
+  return L.marker([lat, lon], { icon });
+}
+
+function createMarkerClusterGroup(classify, color) {
+  const layerOptions = {};
+
+  if (classify) {
+    layerOptions.iconCreateFunction = (cluster) => {
+      const childCount = cluster.getChildCount();
+      const style = `color: white; background-color: ${color};`;
+      return L.divIcon({
+        html: `<div style="${style}"><span>${childCount}</span></div>`,
```

Comment:
Seems like the style attribute is not being added.
<img width="1019" alt="Screen Shot 2019-10-27 at 8 28 04" src="https://user-images.githubusercontent.com/486954/67630662-d7728b80-f893-11e9-83cc-97e4a52c0905.png">

(tested on https://deploy-preview-4278--redash-preview.netlify.com/queries/49#118)

Reply:
@ranbena It worked that way before, so I just kept it. But after your comment I decided that it should be fixed - for details see my comment below :point_down: